### PR TITLE
Silicons have a hit sound and some other fixes and improvements

### LINKED ISSUE #36752: Certain animals are still audible to observers globally when they shouldn't be
A feature added years ago makes it so that simple_animals don't spam every observer's chatlogs, but they still happen for Ian, Poly, Lisa and Sasha because they do certain custom emotes that aren't caught by the code for some reason.

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Fixes #36752 
Attacking silicons now properly plays hitsounds.
Cleaned up item_attack a little. The slime item attack code is now its own code in the slime code (and has been slightly tweaked so that it isn't so long, with some minimal gameplay changes). Moved the mob attack code into attacked_by, which has been turned into a living proc instead of just a carbon proc. Silicons make use of their own variant of this proc in order to play hitsounds because hitsounds were only played in the carbon proc.
Also fixed a bug where critical damage did not apply to slimes for the purpose of how strong the weapon has to be in order to get rid of them.
Slime code has been refactored a little; removed some sleep() and spawn() instances where SStun would wear off afterwards and instead turned SStun into a variable that ticks down every tick. The timings are relatively similar (mostly with Life() timings) and the difference is small enough that nobody will notice. It also helps that repeatedly stunning a slime no longer has a big chance of immediately wearing off because a previous timing was wearing off.
Slimes should no longer be included in the global list of things ghost can hear, just like simple_animals.
Fixed emotes so that slimes can actually do most of their emotes. Previously it checked if the slime was a monkey. This means the slimes should now actually do more emotes over time instead of just "swaying dizzily".
Fixed the "mobs shouldn't play their emotes when no one is near" mechanic even more. It no longer plays emotes even when the type is overridden.
Added a client check before trying to send a to_chat to a mob for a failed emote.
The pronoun proc will now respect neuter/plural genders by removing "%s" from any emote that has it, and will also work for every living mob instead of just humans.
Robots now have a new surrender text.
Removed unused code where attacking a non-carbon with a slime would deal brain damage. It was written in the code but it never kicked in.
Removed redundant ". = TRUE". It has actually been unused code for about 2 years because it was made to only apply to breakable weapons (of which there aren't any in the game, as far as I have glanced over to that) and nothing bad seemed to happen due to that.

:cl:
 * bugfix: Silicons will now properly play hitsounds when hit.
 * bugfix: Fixed a bug where critical hits would not affect the likelihood of getting slimes off of players by using items.
 * bugfix: Slightly reworked how slime stunning works, and in the process fixed a bug where repeatedly stunning a slime would be less effective than intended because the previous stun's "remove the slime's stun" timer cancelled subsequent stuns too early.
 * bugfix: Slimes should no longer be globally audible to observers, just like regular mindless creatures, while someone is near them.
 * bugfix: Fixed a 7-year-old bug where certain slime-exclusive emotes checked if the slime was a monkey. Slimes can now also bounce, jiggle, light up for a bit and vibrate rather than just sway dizzily around.
 * bugfix: Certain pets such as Ian, Lisa, Poly, Sasha etc. will no longer spam deadchat with their actions.
 * tweak: Robots now have a new surrender text.

### CODE REVIEW

File: code/modules/mob/living/living.dm [hunk 1]
```diff
@@ -1917,3 +1917,30 @@ Thanks.
...
+			// if(istype(I, /obj/item/weapon/holder/animal/slime))
+			// 	adjustBrainLoss(force)
+			if(istype(src, /mob/living/carbon/monkey))
+				var/mob/living/carbon/monkey/K = src
+				force = K.defense(force,def_zone)
+			take_organ_damage(force)
+			if (prob(33) && I.force) // Added blood for whacking non-humans too
+				var/turf/location = loc
+				if (istype(location, /turf/simulated))
+					location:add_blood_floor(src)
```
Comment (west3436) [inline code/modules/mob/living/living.dm]: Why are you doing this at runtime rather than compile?

**Comment by west3436** [comment]
Unatomic as all hell but a nice change - please try to keep these more atomic in the future for ease of review and bug isolation.

**Comment by PrimeDSS13** [comment]
good changes but unatomic

**Comment by west3436** [comment]
This could be at least 4 separate PRs. Leaving everything grouped into a single PR will be a nightmare when identifying any bugs that arise. Please open separate ATOMIC PRs to release these and I will merge them.

**Pull request closed without merging**